rotkehlchen/tests/api/test_balances.py

```diff
@@ -303,7 +303,7 @@ def test_query_all_balances_ignore_cache(
         )
         for fn in function_call_counters:
             if fn._mock_wraps == original_binance_query_dict:
-                assert fn.call_count == 3
+                assert fn.call_count == 2
```

Comment:
Why is everything one less query? I saw 2 endpoints added 1 removed no?

Reply:
so this is because these mock function call count are for `api_query_dict`, we were using it for the previous API. But now we're using `api_query_list` for the new API. That is tested [here](https://github.com/rotki/rotki/pull/7507/files#diff-2147b8264bfde8a089c45c3387721b3f48a50b4a8fb94d0721c8d666457b5001R548).